http-server/src/main/java/io/airlift/http/server/DoubleSummaryStats.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.http.server;
+
+import io.airlift.event.client.EventField;
+import io.airlift.event.client.EventType;
+
+import java.util.DoubleSummaryStatistics;
+
+import static java.util.Objects.requireNonNull;
+
+@EventType
```

Comment:
Why does this need to be tagged with `@EventType`?

Reply:
Because, otherwise I get the below error. Is there any other way around this?

```
 Event class [class io.airlift.http.server.HttpRequestEvent] EventField method [public io.airlift.http.server.DoubleSummaryStats io.airlift.http.server.HttpRequestEvent.getResponseContentInterarrivalStats()] return type [class io.airlift.http.server.DoubleSummaryStats] is not supported
  at io.airlift.http.server.HttpServerModule.configure(HttpServerModule.java:76)
```